lib/hammer_cli_csv/content_hosts.rb

```diff
@@ -12,6 +12,7 @@ def self.supported?
       end
 
       option %w(--itemized-subscriptions), :flag, _('Export one subscription per row, only process update subscriptions on import')
+      option %w(--export-columns), 'COLUMN_NAMES', _('Comma separated list of column names to export')
```

Comment:
I'll change this option from _--export-columns_ to simplye _--columns_ to allow the import of only specific columns.